Celeste.Mod.mm/Content/Dialog/English.txt

```diff
@@ -91,6 +91,8 @@
 
 # Sound Test
 	SOUNDTEST_TITLE=	SOUND TEST
+	SOUNDTEST_AUDIOPARAMS_OPEN=	Press Journal to edit Audio Params
```

Comment:
It'd be great if `Journal` could be replaced with the currently active Journal key binding, although it'd require splitting the text into two. Maybe an icon + key binding combo like on the map select screen would do the job too?